source/jormungandr/jormungandr/interfaces/v1/converters_collection_type.py

```diff
@@ -38,7 +38,7 @@
     "pois": "poi", "poi_types": "poi_type",
     "connections": "connection", "vehicle_journeys": "vehicle_journey",
     "disruptions": "disruption", "trips": "trip", "contributors": "contributor",
-    "frames": "frame"}
+    "datasets": "frame"}
```

Comment:
I don't think so, cause it's pb part and we don't want to change it now.
Well we want, but we're in a hurry ;)